pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -169,7 +169,7 @@ self: super: {
   };
 
   # 2024-07-09: rhine 1.4.* needs newer monad-schedule than stackage (and is only consumer)
-  monad-schedule = assert super.monad-schedule.version == "0.1.2.2"; doDistribute self.monad-schedule_0_2;
+  monad-schedule = assert super.monad-schedule.version == "0.1.2.2"; doDistribute self.monad-schedule_0_2_0_1;
```

Comment:
It doesn't look like `rhine-1.4.0.1` depends on `monad-schedule` anymore, so we might be able to drop this override.